src/dataflow/src/source/kafka.rs

```diff
@@ -194,16 +195,13 @@ impl SourceInfo<Vec<u8>> for KafkaSourceInfo {
         }
     }
     /// Returns the number of partitions expected *for this worker*. Partitions are assigned
-    /// round-robin in worker id order
+    /// round-robin in worker id order offset by the hash of the source_id
     /// Ex: a partition count of 4 for 3 workers will assign worker 0 with partitions 0,3,
```

Comment:
`has_partition` would benefit from a descriptive comment like this since that's the key function where the logic happens. The example mentioned here is now out of date

Reply:
comment added!